packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -106,16 +91,6 @@ const getStateStyles = (
       case 'primary':
         return [
           activeStyle,
-          // When the button is active and somehow is disabled it should have
-          // a different color to indicate that it's active but can't receive any actions
-          isDisabled &&
```

Comment:
This styles were never used. If the `isDisabled` prop is true, we return other styles previously in this function.